modules/retail/pages/retail-deploy-terminals-other.adoc

```diff
@@ -212,3 +212,102 @@ Then you can specify a second Salt state with the wireless details for use durin
 The boot phase supports only WPA2 PSK wireless configuration.
 Salt-managed production configuration supports all features supported by all major operating systems.
 ====
+
+
+
+== Deploy Terminals and Auto-Accept Keys
+
+You can configure {productname} to automatically accept the keys of newly deployed terminals.
+This is achieved using Salt grains.
+For more information about automatically accepting Salt grains, see https://docs.saltstack.com/en/latest/topics/tutorials/autoaccept_grains.html.
+
+[IMPORTANT]
+====
+Automatically accepting keys is less secure than manually checking and accepting keys.
+Only use this method on trusted networks.
+====
+
+
+To do this, you first need to configure Saltboot, then configure the {productname} Server.
+
+You can configure Saltboot to send automatically signed grains once and then delete them.
+This method appends the Saltboot configuration to an existing ``initrd``.
+
+You can also choose to keep the automatically signed grains on the Salt client.
+This method includes the configuration file in the image source before the client image is built.
+The auto-signed grain is then stored on the client as a regular Salt grain after booting.
+
+Alternatively, you can configure Saltboot during PXE boot using kernel parameters.
+
+
+
+.Procedure: Configuring Saltboot to Send Auto-Signed Grain Once
+. On the branch server, create a configuration file called [path]``/etc/salt/minion.d/autosign-grains-onetime.conf``.
+. Edit the new configuration file with these details.
+  You can use any value you like as the autosign key:
```

Comment:
wondering whether we should write auto-sign (with a dash)?  Here, and more often.